packages/lib-classifier/src/store/WorkflowStepStore/WorkflowStepStore.js

```diff
@@ -149,6 +162,28 @@ const WorkflowStepStore = types
       })
     }
 
+    function setTaskStrings() {
+      const workflowStrings = self.workflow?.strings
+      self.steps.forEach(function (step) {
+        step.tasks.forEach((task) => {
+          const prefix = `tasks.${task.taskKey}.`
+          const strings = {}
+          Object.entries(workflowStrings).forEach(([key, value]) => {
```

Comment:
Good question. We have come across a live project workflow that had ~27,000 task strings but complex workflows are rare. 

It might be faster to add strings to tasks then split the tasks into steps. I’ll have to remind myself how the store is structured. I can’t remember if tasks are stored directly on steps, or if steps hold references to tasks.